nvflare/app_opt/xgboost/histogram_based_v2/defs.py

```diff
@@ -132,10 +130,10 @@ class SplitMode:
     "horizontal": SplitMode.ROW,
     "v": SplitMode.COL,
     "vertical": SplitMode.COL,
-    "hs": SplitMode.ROW_SECURE,
-    "horizontal_secure": SplitMode.ROW_SECURE,
-    "vs": SplitMode.COL_SECURE,
-    "vertical_secure": SplitMode.COL_SECURE,
+    "hs": SplitMode.ROW,
+    "horizontal_secure": SplitMode.ROW,
+    "vs": SplitMode.COL,
+    "vertical_secure": SplitMode.COL,
 }
 
 SECURE_TRAINING_MODES = {"hs", "horizontal_secure", "vs", "vertical_secure"}
```

Comment:
do we need to have these mappings then?

Reply:
Yes. The mapping is needed to find out the training is vertical or horizontal.